geps/gep-2162/index.md

```diff
@@ -130,21 +131,41 @@ We should treat this guidance as "best effort" because we might have test files
 
 In any case, the conformance tests file names should be meaningful and easy to understand.
 
+#### Conformance report
+
+In order for to verify that the list of features reported are indeed supported by GatewayClass tests will be run based on the features from the GatewayClassStatus.
+If the source of Gateway features are not inferred and manually provided the conformance suite will block the report from being submitted.
+
+#### Reporting Mesh features
+
+As Mesh doesn't have a good place (yet) to populate SupportedFeatures, its features can be tested for compliance by using Conformance Profiles, or manually, providing features for test using `--supported-features` flag.
+Conformance suite will determine if the implementation doesn't support GatewayClass and validate that report is run for the Mesh features and NOT block the report.
```

Comment:
```suggestion
Until mesh features are posted on the Mesh resource CRD, the conformance suite will determine if conformance run is for Mesh features and will NOT block the report if provided manually.
```

Reply:
nit: will be useful to add issue to track the work to support mesh supportedFeatures